src/core_modules/capture-core/components/LockedSelector/LockedSelector.container.js

```diff
@@ -49,11 +48,16 @@ const mapDispatchToProps = (
     onResetAllCategoryOptions: () => {
         dispatch(resetAllCategoryOptionsFromLockedSelector());
     },
-    onOpenNewEventPage: (selectedProgramId, selectedOrgUnitId) => {
```

Comment:
removed these two `selectedProgramId, selectedOrgUnitId` since in the epics we get this information from the redux state. We have been already doing this way. Meaning I didnt refactor this bit. These values must have been forgotten to be removed after some refactoring